Examples/Examples/All Examples/CustomSymbolAnnotationsExample.swift

```diff
@@ -0,0 +1,339 @@
+import UIKit
+import MapboxMaps
+import Turf
+
+@objc(CustomSymbolAnnotationsExample)
```

Comment:
this should immediately precede the view controller class declaration

Reply:
So you are suggesting I move the `enum` and `struct` declarations to the bottom of the file below the `CustomSymbolAnnotationsExample` class?

Reply:
Actually, I believe I know what you meant and moved just the `@objc(CustomSymbolAnnotationsExample)` line to immediately precede the view controller class declaration.